apps/web-giddh/src/app/contact/contact.component.scss

```diff
@@ -557,6 +557,29 @@ button.btn.btn-filter span.icon-cross {
     height: 200px;
 }
 
+.btn-filters {
```

Comment:
Consider moving this code to a global mixin file as it is repeating in various components